<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="REM-L-SINGHH" name="VW_TC.TC0004_login_With_Partner_Customer_Admin" tests="6" failures="1" timestamp="11 Jul 2019 18:44:17 GMT" time="146.920" errors="0">
  <testcase name="logInWithPartnerCustomerAdmin" time="24.085" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin"/>
  <testcase name="headerMandatoryDetails" time="0.347" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin"/>
  <testcase name="footerMandatoryDetails" time="0.220" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin"/>
  <testcase name="dashBoardTabValidation" time="3.329" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin"/>
  <testcase name="alertTabValidation" time="2.305" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin"/>
  <testcase name="testTabValidation" time="116.634" classname="VW_TC.TC0004_login_With_Partner_Customer_Admin">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at PageActions.pom005_VoiceWatch_Tests_Tab.createTestbybutton(pom005_VoiceWatch_Tests_Tab.java:663)
at VW_TC.TC0004_login_With_Partner_Customer_Admin.testTabValidation(TC0004_login_With_Partner_Customer_Admin.java:122)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testTabValidation -->
</testsuite> <!-- VW_TC.TC0004_login_With_Partner_Customer_Admin -->
